src/components/FooterLogo.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import { withClient } from 'cozy-client'
+
+export class FooterLogo extends React.Component {
+  state = {
+    logos: []
+  }
+
+  async componentDidMount() {
+    const logos = await this.fetchHomeLogos()
+    this.setState({ logos })
+  }
+
+  async fetchHomeLogos() {
+    try {
+      const context = await this.props.client
+        .getStackClient()
+        .fetchJSON('GET', '/settings/context')
+      const logos = get(context, 'data.attributes.home_logos', [])
+      return logos
+    } catch (error) {
+      return []
+    }
+  }
+
+  render() {
+    const { logos } = this.state
+    const { client } = this.props
+    const rootURL = client.getStackClient().uri
```

Comment:
The concatenation of rootURL and the logos URL could be done directly in the fetchHomeLogos, which would be put less responsibility and require less client knowledge from the component.